app/extensions/safe/test/app/webviewPreload.spec.ts

```diff
@@ -16,6 +17,34 @@ jest.mock( 'extensions/safe/ffi/authenticator', () => jest.fn() );
 
 jest.mock( '@maidsafe/safe-node-app', () => jest.fn() );
 
+const pendingCalls = {};
+
+const createRemoteCall = ( functionName, passedStore ) => {
```

Comment:
Can we not pull this in from `webPreload`? Or mock it? Seems overkill / dangerous to be replicating the func here.